cni/network/network.go

```diff
@@ -821,10 +799,29 @@ func (plugin *NetPlugin) createEndpointInternal(opt *createEndpointInternalOpt)
 		return epInfo, plugin.Errorf(err.Error())
 	}
 
+	epInfos := []*network.EndpointInfo{&epInfo}
+	// get multitenant/secondary interface info
+	for _, multitenantResult := range opt.ipamAddResult.cniResults {
```

Comment:
can we name it as secondaryResults to avoid confusion with vlan multitenancy

Reply:
yup will update